IsraelHiking.Web/src/application/components/dialogs/intro-dialog.component.ts

```diff
@@ -6,12 +6,29 @@ import { NgRedux } from "../../reducers/infra/ng-redux.module";
 import { ResourcesService } from "../../services/resources.service";
 import { ApplicationState } from "../../models/models";
 import { ConfigurationActions } from "application/reducers/configuration.reducer";
+
+import { AnimationItem } from 'lottie-web';
+import { AnimationOptions } from 'ngx-lottie';
 @Component({
     selector: "intro-dialog",
-    templateUrl: "./intro-dialog.component.html"
+    templateUrl: "./intro-dialog.component.html",
+    styleUrls: ["./intro-dialog.component.scss"]
 })
 export class IntroDialogComponent extends BaseMapComponent {
 
+    lottieLanguage: AnimationOptions = {
```

Comment:
I'm not sure, but maybe you import the json file itself and use its content to solve the delay you noticed.
I'll take a look at it and let you know...